azkaban-common/src/test/java/azkaban/executor/JdbcExecutorLoaderTest.java

```diff
@@ -873,6 +874,29 @@ public void testFetchActiveFlowByExecId() throws Exception {
     Assert.assertEquals(flow1.getVersion(), execFlow1.getVersion());
   }
 
+  @Test
+  public void testFetchRecentlyFinishedFlows() throws Exception {
+    if (!isTestSetup()) {
```

Comment:
Yes, in order to run all these tests in JdbcExecutorLoaderTest class locally, we need to modify the database, user and password to match our local mysql setup. In Travis, all the tests will be disabled because DB is not set up. Once we change it to in-memory database later, this JdbcExecutorLoaderTest class will be refactored and the tests won't have this setup limitation.